tests/unit/utils/Utils.h

```diff
@@ -0,0 +1,15 @@
+#ifndef __UTILS_H__
+#define __UTILS_H__
+
+/*
```

Comment:
Yes, I wrote it. However, I need to admit that I took inspiration from some C++ guide (like this one http://www.cplusplus.com/reference/cstring/strstr/) and from some StackOverflow answers (for the lambda thing).
Should I insert some links as acknowledgement?